util/http.go

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 1999-2020 Alibaba Group Holding Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package util
+
+import "net/url"
+
+func EncodingParams(params map[string]string) map[string]string {
```

Comment:
maybe it's better way.
```
 func EncodingParams(params map[string]string) {
	for param, value := range params {
		params [param] = url.QueryEscape(value)
 	}
 }
```

Reply:
EncodingParams func should be used in http_agent#Request func

Reply:
You are right, it's a mistake,  maybe java has deeply affected me. 
And now, it run  expected